test/ctr_unpause.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
```

Comment:
nit: can we combine the files for pause/unpause?

Reply:
done